### PR TITLE
docs:add tuple data type

### LINKED ISSUE #5639: Documentation of tuple types
Seems there is no document of tuple types in databend.rs

image

### PR DESCRIPTION
**Pull request by wubx**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

add data type tuple

Closes #5639

### CODE REVIEW

File: docs/doc/13-sql-reference/10-data-types/41-data-type-tuple.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+title: Tuple
+description:  Tuple is a collection of ordered, immutableTypes can hold any other data types.
+---
+
+## Tuple Data Types
+| Name    |  Aliases  | Build From Values        | Description
+|---------|-----------|--------------------------|----------------
+| TUPLE   |           | ('2023-02-14 08:00:00','Valentine's Day') | Collection of ordered,immmutable. Like an immutable variant, which requires the type of each element to be declared before being used.
+
+## Tuple Data Types
```
Comment (sundy-li) [inline docs/doc/13-sql-reference/10-data-types/41-data-type-tuple.md:11]: Duplicated

File: docs/doc/13-sql-reference/10-data-types/41-data-type-tuple.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+---
+title: Tuple
+description:  Tuple is a collection of ordered, immutableTypes can hold any other data types.
+---
+
+## Tuple Data Types
+| Name    |  Aliases  | Build From Values        | Description
```
Comment (sundy-li) [inline docs/doc/13-sql-reference/10-data-types/41-data-type-tuple.md:7]: What's `Build From Values`?

File: docs/doc/13-sql-reference/10-data-types/index.md [hunk 3]
```diff
@@ -37,3 +37,4 @@ Databend supports three Semi-structured types: ARRAY, OBJECT and VARIANT.
 |---------|--------------|-------------------------|----------------
 | ARRAY   |              | [1,2,3]                 | Zero-based indexed list, each value can have difference data type.
 | VARIANT |  JSON        | [1,{"a":1,"b":{"c":2}}] | Collection of elements of different data types., including ARRAY and OBJECT.
+| TUPLE   |              | ('2023-02-14 08:00:00','Valentine's Day') | Collection of ordered,immmutable. Like an immutable variant, which requires the type of each element to be declared before being used.
```
Comment (sundy-li) [inline docs/doc/13-sql-reference/10-data-types/index.md:40]: It's not `Semi-structured `

File: docs/doc/13-sql-reference/10-data-types/41-data-type-tuple.md (hunk 2 above)
Comment (wubx) [reply docs/doc/13-sql-reference/10-data-types/41-data-type-tuple.md]: change build from values to  values

File: docs/doc/13-sql-reference/10-data-types/41-data-type-tuple.md (hunk 1 above)
Comment (wubx) [reply docs/doc/13-sql-reference/10-data-types/41-data-type-tuple.md]: remove it

File: docs/doc/13-sql-reference/10-data-types/index.md (hunk 3 above)
Comment (wubx) [reply docs/doc/13-sql-reference/10-data-types/index.md]: remove semi-structed

File: docs/doc/13-sql-reference/10-data-types/index.md [hunk 4]
```diff
@@ -37,4 +37,4 @@ Databend supports three Semi-structured types: ARRAY, OBJECT and VARIANT.
 |---------|--------------|-------------------------|----------------
 | ARRAY   |              | [1,2,3]                 | Zero-based indexed list, each value can have difference data type.
 | VARIANT |  JSON        | [1,{"a":1,"b":{"c":2}}] | Collection of elements of different data types., including ARRAY and OBJECT.
-| TUPLE   |              | ('2023-02-14 08:00:00','Valentine's Day') | Collection of ordered,immmutable. Like an immutable variant, which requires the type of each element to be declared before being used.
+| TUPLE   |              | ('2023-02-14 08:00:00','Valentine's Day') | Collection of ordered,immmutable, which requires the type of each element to be declared before being used.
```
Comment (sundy-li) [inline docs/doc/13-sql-reference/10-data-types/index.md:40]: It's not a category of semi struct.

Comment (wubx) [reply docs/doc/13-sql-reference/10-data-types/index.md]: add " nested data type"

**Review by sundy-li — APPROVED** [review]

**Pull request merged**